<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartGet</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="newMission">
          <property name="text">
           <string>新任务</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startDaemon">
          <property name="text">
           <string>开启Daemon</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>文件名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>大小</string>
         </property>
         <property name="textAlignment">
          <set>AlignRight|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>已完成</string>
         </property>
         <property name="textAlignment">
          <set>AlignRight|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>%</string>
         </property>
         <property name="textAlignment">
          <set>AlignRight|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>速度</string>
         </property>
         <property name="textAlignment">
          <set>AlignRight|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>节点数</string>
         </property>
         <property name="textAlignment">
          <set>AlignRight|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string> 文件</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>首选项</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
